src/commands/retention-policies/assign.js

```diff
@@ -42,6 +50,31 @@ RetentionPoliciesAssignCommand.flags = {
 	'assign-to-id': flags.string({
 		description: 'The ID of the content to assign the retention policy to',
 	}),
+	'filter-fields': flags.string({
+		description: ' Metadata fields to filter against, if assigning to a metadata template.' +
+			'Allow properties: field, value. Example: field=foo,value=bar',
+		multiple: true,
+		parse(input) {
+			const filter = {};
+			input.split(',').forEach((pair) => {
+				const [key, value] = pair.split('=');
+				switch (key) {
+					case 'field':
+						filter.field = value;
+						break;
+					case 'value':
+						filter.value = value;
+						break;
+					default:
+							throw new BoxCLIError(`Unknown property for signer: ${key}`);
```

Comment:
One tab too much, please fix this indentation and an Exception message as it's not related to sign api